keps/sig-windows/1001-windows-cri-containerd/README.md

```diff
@@ -100,40 +100,49 @@ Additionally, users could choose to run with only CRI-ContainerD instead of Dock
 ### Non-Goals
 
 - Running Linux containers on Windows nodes. This would be addressed as a separate KEP since the use cases are different.
-- Deprecating `dockershim`. This is out of scope for this KEP. The effort to migrate that code out of tree is in [KEP PR 866](https://github.com/kubernetes/enhancements/pull/866) and deprecation discussions will happen later.
+- Deprecating `dockershim`. This is out of scope for this KEP. The effort to migrate that code out of tree is in [KEP PR 866](https://github.com/kubernetes/enhancements/pull/866) and [deprecation](https://github.com/kubernetes/kubernetes/pull/94624) discussions are in flight.
 
 ## Proposal
 
 ### User Stories
 
-#### Improving Kubernetes integration for Windows Server containers
+#### Increasing the parity between Kubernetes on Linux and Windows Server containers
 
 Moving to the new Windows HCSv2 platform and ContainerD would allow Kubernetes to add support for:
 
 - Mounting single files, not just folders, into containers
 - Termination messages (depends on single file mounts)
 - /etc/hosts (c:\windows\system32\drivers\etc\hosts) file mapping
 
-#### Improved isolation and compatibility between Windows pods using Hyper-V 
+#### Increasing the forward-compatibility and security of Windows pods by leveraging Hyper-V
 
-Hyper-V enables each pod to run within it’s own hypervisor partition, with a separate kernel. This means that we can build forward-compatibility for containers across Windows OS versions - for example a container built using Windows Server 1809, could be run on a node running Windows Server 1903. This pod would use the Windows Server 1809 kernel to preserve full compatibility, and other pods could run using either a shared kernel with the node, or their own isolated Windows Server 1903 kernels. Containers requiring 1809 and 1903 (or later) cannot be mixed in the same pod, they must be deployed in separate pods so the matching kernel may be used. Running Windows Server version 1903 containers on a Windows Server 2019/1809 host will not work.
+Currently, the only way to run Hyper-V isolation in Kubernetes is to use the deprecated `experimental` isolation type, which was added in v1.10.
```

Comment:
There is a PR in open now to remove this flag (https://github.com/kubernetes/kubernetes/pull/94531) if you want to link to it.
Also, it might be good to call our the limitations here which are that Docker support for Hyper-V isolated containers is limited to a single Container per Pod - which is not very useful.